examples/deployment-readwriteonce.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-deployment-nfs-rwo
+spec:
+  accessModes:
+    - ReadWriteOnce

```

Comment:
I don't think we need two similar yaml files, one for `ReadWriteOnce` and one for `ReadWriteMany`, I would suggest only keep `ReadWriteMany`, and add comment in that yaml file, e.g. `Available values: ReadWriteMany, ReadWriteOnce, ReadOnlyMany`, actually all `ReadWriteMany` accessModes would also those 3 modes

Reply:
OK, agree. Will make the change.